### PR TITLE
feat: change aside

### PR DESCRIPTION
**Pull request by eszczepan**
## Summary
This PR changes aside links.

## Related issue
Ref: 

## Test plan
Visit ... and check ...

## Screenshots

### CODE REVIEW

File: src/App/Blog/index.jsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import './styles.css';
+
+export const Blog = () => {
+  //BEM methodology
```
Comment (eszczepan) [inline src/App/Blog/index.jsx:4]: Please remove this comment

File: src/App/Calendar/index.jsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import './styles.css';
+
+export const Calendar = () => {
```
Comment (eszczepan) [inline src/App/Calendar/index.jsx:3]: Nice change 🎉

**Review by eszczepan — COMMENTED** [review]
Nicely done 💯 🎖️

**Review by caffeine108 — APPROVED** [review]
Well done 👍

**Pull request merged**